azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -137,6 +138,12 @@ public void succeedJob(final Props generatedProperties) {
     }
   }
 
+  public void ignoreCancel() {
+    synchronized (this) {
+      this.ignoreCancel = true;
```

Comment:
ok, i see. yeah you are right, let's keep it for now to be consistent and refactor it in a later commit.